coralproject/talk
# Add missing translation keys

## What does this PR do?
- Updates missing core translation keys for all current languages
- Updates missing plugin translation keys for all current languages

## How do I test this PR?

- Change between languages in local storage
- See that the page is being translated properly
